vsintegration/src/FSharp.LanguageService/ProjectSitesAndFiles.fs

```diff
@@ -101,7 +109,75 @@ type private ProjectSiteOfSingleFile(sourceFile) =
         override this.AssemblyReferences() = [||]
         
     override x.ToString() = sprintf "ProjectSiteOfSingleFile(%s)" sourceFile
+ 
+type CommandLineOptions (isEmpty:bool) =
+
+    let commandLineOptions = new ConcurrentDictionary<ProjectId, string[]*string[]*string[]>()
+    static let emptyCommandLineOptions = lazy new CommandLineOptions(true)
```

Comment:
There is no point in making such a trivial object allocation lazy - just remove the ``lazy`